ddsketch/ddsketch.py

```diff
@@ -201,6 +200,32 @@ def copy(self, sketch):
         self.count = sketch.count
         self._sum = sketch.sum
 
+    def to_proto(self):
+        """serialize to protobuf"""
+        return pb.DDSketch(
+            mapping=self.mapping.to_proto(),
+            positiveValues=self.store.to_proto(),
+            negativeValues=self.negative_store.to_proto(),
+            zeroCount=self.zero_count,
+        )
+
+    @classmethod
+    def from_proto(cls, proto):
+        """deserialize from protobuf
+
+        N.B., The current protobuf loses any min/max/sum/avg information.
+        """
+        mapping = KeyMapping.from_proto(proto.mapping)
+        negative_store = DenseStore.from_proto(proto.negativeValues)
+        store = DenseStore.from_proto(proto.positiveValues)
```

Comment:
Should the stores always be `DenseStore`s after deserializing?

Reply:
I think so, no? At least in Python the only other options are Collapsing ones which aren't really supported by what's in the proto

Reply:
You're right - I thought the java version was setting the `maxNumBins` as well, but it's not. I'll change my code.